Resources/doc/installation.md

```diff
@@ -69,6 +69,35 @@ sulu_article:
     prefix: /admin/articles
 ```
 
+### Webspace behavior
+
+When you have more than one webspace in your Sulu project you must provide a default main webspace and optional
+you can also provide default additional webspaces.
+
+#### Functionality
+
+An article is delivered only on the main webspace.
```

Comment:
Normally an article will be delivered only for the main webspace. Further an article can be delivered for the additional webspaces which are configured. To prevent duplicate content issues a `canonical tag` with the url for the main webspace is inclued in the HTML head.